terraform/instances.tf

```diff
@@ -23,25 +23,10 @@ resource "aws_instance" "api" {
     agent       = "${var.connection_agent}"
   }
 
-  tags {
-    Name          = "builder-api-${count.index}"
-    X-Contact     = "The Habitat Maintainers <humans@habitat.sh>"
-    X-Environment = "${var.env}"
-    X-Application = "builder"
-    X-ManagedBy   = "Terraform"
-  }
-}
-
-resource "null_resource" "api_provision" {
-  triggers {
-    ebs_volume = "${aws_volume_attachment.api.id}"
-  }
-
-  connection {
```

Comment:
Ok It's been a hot minute since I've written any terraform templates. Did this `connection` resource get removed intentionally?

Reply:
Yep we no longer need a separate "api_provision" resource.. everything is now merged into the "api" resource.